libraries/botbuilder/src/botFrameworkAdapter.ts

```diff
@@ -434,6 +434,27 @@ export class BotFrameworkAdapter extends BotAdapter implements IUserTokenProvide
         return (await client.botSignIn.getSignInUrl(finalState, { channelId: context.activity.channelId }))._response.bodyAsText;
     }
 
+    /** 
+     * Retrieves the token status for each configured connection for the given user.
+     * @param context Context for the current turn of conversation with the user.
+     * @param userId The user Id for which token status is retrieved.
+     * @param includeFilter Optional comma seperated list of connection's to include. Blank will return token status for all configured connections.
+     * @returns Array of TokenStatus
+     * */ 
+    
+    public async getTokenStatus(context: TurnContext, userId?: string, includeFilter?: string ): Promise<TokenStatus[]>
+    {
+        if (!userId && (!context.activity.from || !context.activity.from.id)) {
+            throw new Error(`BotFrameworkAdapter.getTokenStatus(): missing from or from.id`);
+        }
+        this.checkEmulatingOAuthCards(context);
+        ! userId? userId = context.activity.from.id : userId;
```

Comment:
Can you use a more traditional if on this, the !userId? is a confusing bit of syntax for humans.

Reply:
Sure! Syntax updated 😄